content/x-ray/x-ray-daemon.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Deploy X-Ray DaemonSet"
+date: 2018-11-177T08:30:11-07:00
+weight: 11
+draft: false
+---
+
+Now that we have modified the IAM role for the worker nodes to permit write operations to the X-Ray service, we are going to deploy the X-Ray [DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) to the EKS cluster. The X-Ray daemon will be deployed to each worker node in the EKS cluster. For reference, see the [example implementation](https://github.com/rnzsgh/eks-workshop-x-ray-daemon) used in this module.
```

Comment:
Instead of referencing/using the files from another repository, bring the files into this repository and deploy similar to https://eksworkshop.com/logging/deploy/